README.md

```diff
@@ -36,33 +36,59 @@ $ ansible-playbook -i localhost, --connection=local \
 
 ## Deploy new cluster + KubeVirt
 
-
 ### Requirements
+
+Make sure that you have Ansible 2.3.1.
```

Comment:
After #63 and #53 can't we use ansible 2.4+?

Reply:
We are already using 2.4.2, this is leftover .... I will update it. Thanks for catch.